### PR TITLE
feat: move client_id & client_secret to POST body

### PR DESCRIPTION
**Pull request by maricn**
Corresponding changelog: https://dash.readme.com/project/personio/v1.0/changelog/auth-api-moved-client_secret-and-client_id-to-post-body

### CODE REVIEW

File: personio-auth-api.yaml [hunk 1]
```diff
@@ -17,17 +17,20 @@ paths:
...
-          type: string
-          description: Client id of the downloaded credentials file
-        - name: client_secret
-          in: query
-          required: true
-          type: string
-          description: Client secret of the downloaded credentials file
+      requestBody:
+        content:
+          application/json:
```
Comment (maricn) [inline personio-auth-api.yaml:22]: or should this be multipart form?

Comment (maricn) [reply personio-auth-api.yaml:22]: in practice, both work..

Comment (dinomite94) [reply personio-auth-api.yaml:22]: I personally would go for the `requestBody` approach that you already have here

Comment (maricn) [reply personio-auth-api.yaml:22]: actually, i can put both down, and then it should be rendered as "either or"

**Review by dinomite94 — APPROVED** [review]

**Pull request merged**